examples/aggregator/module/rust/src/lib.rs

```diff
@@ -35,159 +35,142 @@ pub mod proto {
-                        *entry = None;
-                        if let Err(err) = self.report(bucket, aggregated_data) {
-                            // Backend report errors are not returned to the clients.
-                            error!("Backend report error: {:?}", err);
-                        }
-                    }
-                }
-                None => return Err(format!("Outdated bucket: {}", bucket)),
-            },
-            None => {
-                let mut aggregator = ThresholdAggregator::<SparseVector>::new(SAMPLE_THRESHOLD);
-                aggregator.submit(svec);
-                self.aggregators.insert(bucket, Some(aggregator));
-            }
-        }
-        Ok(())
-    }
+impl Router {
+    fn create(config: &Config) -> anyhow::Result<Self> {
+        // Create a gRPC client pseudo-Node with a less restrictive label than the current Node.
+        // In particular, the confidentiality component of the current Node label includes the
+        // "aggregator" WebAssembly hash, which is declassified as part of the gRPC client
+        // Node creation. This is only allowed if the current Node actually has the
+        // appropriate capability (i.e. the correct WebAssembly module hash) as specified by
+        // the label component being removed, as set by the client.
+        // TODO(#814): Uncomment and use correct confidentiality labels.
+        // let label = Label {
+        //     confidentiality_tags: vec![Tag {
+        //         tag: Some(tag::Tag::TlsEndpointTag(TlsEndpointTag {
+        //             certificate_subject_alternative_name: "google.com".to_string(),
+        //         })),
+        //     }],
+        //     integrity_tags: vec![],
+        // };
+        let grpc_client = oak::grpc::client::Client::new(
+            "grpc_client",
+            &oak::node_config::grpc_client(&config.grpc_client_connect_address),
+            &Label::public_untrusted(),
+        )
+        .context("Couldn't create gRPC client")?;
+
+        // Create an Aggregator Node.
+        let (aggregator_sender, aggregator_receiver) = oak::io::channel_create::<
+            Either<AggregatorInit, oak::grpc::Invocation>,
+        >(
+            "Aggregator init", &Label::public_untrusted()
+        )
+        .context("Couldn't create channel to aggregator node")?;
+        oak::node_create(
+            "aggregator",
+            &oak::node_config::wasm("app", "aggregator"),
+            &Label::public_untrusted(),
+            aggregator_receiver.handle,
+        )
+        .expect("Couldn't create aggregator node");
+        oak::channel_close(aggregator_receiver.handle.handle).expect("Couldn't close receiver channel");
+
+        // Send the initialization message to Aggregator Node containing a gRPC server invocation
+        // receiver and a gRPC client invocation sender.
+        debug!("Sending the initialization message to handler Node");
```

Comment:
Does handler == aggregator == Aggregator? If so, could you make the messages and comments more consistent?